static_files/main.js

```diff
@@ -0,0 +1,22 @@
+
+const  Menu = document.getElementById("HiddenMenu");
+
+document
+  .querySelector("#ShowMenuButton")
+  .addEventListener("click", MakeVisible);
+
+
+function MakeVisible (){
```

Comment:
To make code clearer, how about of change the name of this function to a name that nake easier the reading? For example, `showMenu`?
Ps: functions names should use a camelCase (lowerCamelCase)

![image](https://github.com/catho/ssr-quantum-simulator/assets/3389749/04d3c3e0-cc9a-4288-a6aa-d86ac016b34c)

ih this case, makeVisible is a method.

https://coodesh.com/blog/dicionario/o-que-e-camelcase/#:~:text=Voc%C3%AA%20j%C3%A1%20ouviu%20falar%20em,com%20exce%C3%A7%C3%A3o%20do%20primeiro%20termo.